### ISSUE TITLE
[Bug]:  Firefox issue with new UI - "We couldn't load the latest Netdata UI"

### ISSUE THREAD

**Issue by emsofo** [netdata/netdata#15987]
### Bug description

I have an issue with 1.42.2 running as plugin for OPNSense - I can load the interface under chrome, but under firefox 117 it's stuck on "Welcome to Netdata - Loading latest Netdata UI..." - I can load the "old" interface fine - and if hit refresh many times - I see the error: "We couldn't load the latest Netdata UI"

### Expected behavior

When opening the UI using Chrome browser - the new UI works fine - doing the same with Firefox 117 - I get the error "We couldn't load the latest Netdata UI"

### Steps to reproduce

1.  Open Firefox browser and go to https://192.168.30.1:3000 (which is my OPNSense router running the netdata instance)
2.  get the error "Welcome to Netdata - Loading latest Netdata UI...    Or you can load the [old single node dashboard](javascript:goToOld('/v1');) or a [local copy](javascript:goToOld('/v2');) of Netdata UI
3. Doing the same with chrome - the new UI works fine
...

### Installation method

other

### System info

```shell
uname -a; uname -K
FreeBSD FoxOPN.fritz.box 13.2-RELEASE-p3 FreeBSD 13.2-RELEASE-p3 stable/23.7-n254799-01e2d59d337 SMP amd64
1302001
```

### Netdata build info

```shell
[rfox@FoxOPN ~]$ netdata -W buildinfo
Packaging:
    Netdata Version ____________________________________________ : v1.42.2
    Installation Type __________________________________________ : custom
    Package Architecture _______________________________________ : unknown
    Package Distro _____________________________________________ : unknown
    Configure Options __________________________________________ :  '--disable-h2o' '--disable-unit-tests' '--disable-plugin-cups' '--enable-dbengine' '--disable-plugin-freeipmi' '--enable-lto' '--enable-ml' '--prefix=/usr/local' '--localstatedir=/var' '--mandir=/usr/local/man' '--disable-silent-rules' '--infodir=/usr/local/share/info/' '--build=amd64-portbld-freebsd13.2' 'build_alias=amd64-portbld-freebsd13.2' 'CC=cc' 'CFLAGS=-O2 -pipe  -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing ' 'LDFLAGS= -Wl,-rpath,/usr/local/lib -fstack-protector-strong -L/usr/local/lib ' 'LIBS=' 'CPPFLAGS=-D_WANT_VMMETER -isystem /usr/local/include' 'CXX=c++' 'CXXFLAGS=-O2 -pipe -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -isystem /usr/local/include ' 'PKG_CONFIG=pkgconf' 'PKG_CONFIG_LIBDIR=/usr/obj/usr/ports/net-mgmt/netdata/work/.pkgconfig:/usr/local/libdata/pkgconfig:/usr/local/share/pkgconfig:/usr/libdata/pkgconfig' 'CPP=cpp'
Default Directories:
    User Configurations ________________________________________ : /usr/local/etc/netdata
    Stock Configurations _______________________________________ : /usr/local/lib/netdata/conf.d
    Ephemeral Databases (metrics data, metadata) _______________ : /var/cache/netdata
    Permanent Databases ________________________________________ : /var/db/netdata
    Plugins ____________________________________________________ : /usr/local/libexec/netdata/plugins.d
    Static Web Files ___________________________________________ : /usr/local/share/netdata/web
    Log Files __________________________________________________ : /var/log/netdata
    Lock Files _________________________________________________ : /var/db/netdata/lock
    Home _______________________________________________________ : /var/db/netdata
Operating System:
    Kernel _____________________________________________________ : FreeBSD
    Kernel Version _____________________________________________ : 1302001
    Operating System ___________________________________________ : FreeBSD
    Operating System ID ________________________________________ : FreeBSD
    Operating System ID Like ___________________________________ : FreeBSD
    Operating System Version ___________________________________ : 13.2-RELEASE-p3
    Operating System Version ID ________________________________ : none
    Detection __________________________________________________ : uname
Hardware:
    CPU Cores __________________________________________________ : 4
    CPU Frequency ______________________________________________ : unknown
    CPU Architecture ___________________________________________ : 8550580224
    RAM Bytes __________________________________________________ : 24744353792
    Disk Capacity ______________________________________________ : amd64
    Virtualization Technology __________________________________ : kvm
    Virtualization Detection ___________________________________ : sysctl
Container:
    Container __________________________________________________ : unknown
    Container Detection ________________________________________ : none
    Container Orchestrator _____________________________________ : none
    Container Operating System _________________________________ : none
    Container Operating System ID ______________________________ : none
    Container Operating System ID Like _________________________ : none
    Container Operating System Version _________________________ : none
    Container Operating System Version ID ______________________ : none
    Container Operating System Detection _______________________ : none
Features:
    Built For __________________________________________________ : FreeBSD
    Netdata Cloud ______________________________________________ : YES
    Health (trigger alerts and send notifications) _____________ : YES
    Streaming (stream metrics to parent Netdata servers) _______ : YES
    Replication (fill the gaps of parent Netdata servers) ______ : YES
    Streaming and Replication Compression ______________________ : YES (lz4)
    Contexts (index all active and archived metrics) ___________ : YES
    Tiering (multiple dbs with different metrics resolution) ___ : YES (5)
    Machine Learning ___________________________________________ : YES
Database Engines:
    dbengine ___________________________________________________ : YES
    alloc ______________________________________________________ : YES
    ram ________________________________________________________ : YES
    map ________________________________________________________ : YES
    save _______________________________________________________ : YES
    none _______________________________________________________ : YES
Connectivity Capabilities:
    ACLK (Agent-Cloud Link: MQTT over WebSockets over TLS) _____ : YES
    static (Netdata internal web server) _______________________ : YES
    h2o (web server) ___________________________________________ : NO
    WebRTC (experimental) ______________________________________ : NO
    Native HTTPS (TLS Support) _________________________________ : YES
    TLS Host Verification ______________________________________ : YES
Libraries:
    LZ4 (extremely fast lossless compression algorithm) ________ : YES
    zlib (lossless data-compression library) ___________________ : YES
    Judy (high-performance dynamic arrays and hashtables) ______ : YES (bundled)
    dlib (robust machine learning toolkit) _____________________ : YES (bundled)
    protobuf (platform-neutral data serialization protocol) ____ : YES (system)
    OpenSSL (cryptography) _____________________________________ : YES
    libdatachannel (stand-alone WebRTC data channels) __________ : NO
    JSON-C (lightweight JSON manipulation) _____________________ : YES
    libcap (Linux capabilities system operations) ______________ : NO
    libcrypto (cryptographic functions) ________________________ : YES
    libm (mathematical functions) ______________________________ : YES
    jemalloc ___________________________________________________ : NO
    TCMalloc ___________________________________________________ : NO
Plugins:
    apps (monitor processes) ___________________________________ : YES
    cgroups (monitor containers and VMs) _______________________ : NO
    cgroup-network (associate interfaces to CGROUPS) ___________ : NO
    proc (monitor Linux systems) _______________________________ : NO
    tc (monitor Linux network QoS) _____________________________ : NO
    diskspace (monitor Linux mount points) _____________________ : NO
    freebsd (monitor FreeBSD systems) __________________________ : YES
    macos (monitor MacOS systems) ______________________________ : NO
    statsd (collect custom application metrics) ________________ : YES
    timex (check system clock synchronization) _________________ : YES
    idlejitter (check system latency and jitter) _______________ : YES
    bash (support shell data collection jobs - charts.d) _______ : YES
    debugfs (kernel debugging metrics) _________________________ : NO
    cups (monitor printers and print jobs) _____________________ : NO
    ebpf (monitor system calls) ________________________________ : NO
    freeipmi (monitor enterprise server H/W) ___________________ : NO
    nfacct (gather netfilter accounting) _______________________ : NO
    perf (collect kernel performance events) ___________________ : NO
    slabinfo (monitor kernel object caching) ___________________ : NO
    Xen ________________________________________________________ : NO
    Xen VBD Error Tracking _____________________________________ : NO
Exporters:
    AWS Kinesis ________________________________________________ : NO
    GCP PubSub _________________________________________________ : NO
    MongoDB ____________________________________________________ : NO
    Prometheus (OpenMetrics) Exporter __________________________ : YES
    Prometheus Remote Write ____________________________________ : NO
    Graphite ___________________________________________________ : YES
    Graphite HTTP / HTTPS ______________________________________ : YES
    JSON _______________________________________________________ : YES
    JSON HTTP / HTTPS __________________________________________ : YES
    OpenTSDB ___________________________________________________ : YES
    OpenTSDB HTTP / HTTPS ______________________________________ : YES
    All Metrics API ____________________________________________ : YES
    Shell (use metrics in shell scripts) _______________________ : YES
Debug/Developer Features:
    Trace All Netdata Allocations (with charts) ________________ : NO
    Developer Mode (more runtime checks, slower) _______________ : NO
[rfox@FoxOPN ~]$
```

**Comment by Trufax** [comment]
i can confirm the issue on Firefox 118.0.2 still only shows the Welcome to Netdata.

**Comment by emsofo** [comment]
I tested further - Tried "Safe Mode" - and turned off all extensions - didn't help . . . but if I start with a freshly installed Firefox 
and DON'T login to the Firefox Account - it works ?!? So it has something to do with a logged in Firefox account maybe?

**Comment by Trufax** [comment]
i dont use the Firefox Account and it still does not work.

**Comment by ilyam8** [comment]
There was an issue with `registry?action=hello` returning 400 when Do Not Track was enabled. Fixed in #16212.

**Comment by emsofo** [comment]
That was it!!! Turning off DNT fixed the issue . . . go figure!   Thx!

**Comment by ilyam8** [comment]
Closing as it was fixed in #16212.

**Comment by gitaarik** [comment]
I just ran into this issue, but it was because a Firefox privacy plugin I use, uMatrix, by default blocks loading scripts from third-party domains, like `netdata.cloud` in this case. Just putting it here for documentation sake ;)
